smoke_test/steps/prisoner_page.rb

```diff
@@ -1,11 +1,11 @@
 module SmokeTest
   module Steps
     class PrisonerPage < BaseStep
-      PAGE_PATH = '/prisoner'
+      PAGE_PATH = %r{/prisoner$}
```

Comment:
I don't think this is correct - should expect the url to be something like: `https://www.prisonvisits.service.gov.uk/prisoner`